src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+
+export const NormalPackageContainer = styled.div<{ left: boolean }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 3vh;
+  height: 90%;
+  width: 280px;
+  background: hsl(0, 100%, 100%);
+  ${(props) => (props.left ? 'border-radius: 10px 0px 0px 10px;' : 'border-radius: 0px 10px 10px 0px;')}
+  @media (max-width: 768px) {
+    width: 90%;
+    margin-top: 3vh;
+    border-radius: 10px;
+  }
+`
+
+export const PackageHeader = styled.div`
+  text-align: center;
+  font-size: 20px;
+  font-weight: 800;
+  margin-top: 3vh;
+  color: hsl(233, 13%, 49%);
+`
+
+export const NormalPriceContainer = styled.div`
+  position: relative;
+  z-index: 999;
```

Comment:
hhahaa 1 or 2 would probably be sufficient

Reply:
Resolved in e8c8b98